Concorde.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "Concorde"
-  s.version          = "0.1.0"
+  s.version          = "0.1.1"
```

Comment:
Allright, I didn't know podspecs could be updated therefore I created a new version.